### ISSUE TITLE
QUESTION: Ensuring the latest UH-VPN version

### ISSUE THREAD

**Issue by sunny75016** [ultrahorizon/UH-VPN-Docs#46]
I notice today that the routine "apt update && apt upgrade" has updated the uh-vpn-server. However, there is an inconsistency and I cannot be sure that the latest version has been installed. The result of the apt list command is below.
root@vpn:~# apt list uh-vpn-server
Listing... Done
**uh-vpn-server/focal,now 1.2.0-3 all [installed]**
root@vpn:~#

Google search shows a version number 2.0.0 - but that is wrong and seems to relate to the documentation.

In short, it will be nice if you can help in ascertaining the version number. Perhaps a changelog page on the website or github will equally serve this purpose.

**Comment by AnthonyWharton** [comment]
Please could you format your command outputs as code listings. For help on this [refer to GitHub documentation](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#quoting-code).

**Comment by jwsi** [comment]
Hi @sunny75016,

To follow on from Anthony's reponse, you can also find the latest version from the command line by executing:

`apt-cache policy uh-vpn-server`

This will tell you what version is currently installed and if there is an update available.

If you can link us to the Google result then we can look into that in greater detail.

**Comment by sunny75016** [comment]
Ok. Thanks. Your solution works. I am able to reproduce the version 2.0.0 in a Startpage result. See screen snip.
![uh-screenshot](https://user-images.githubusercontent.com/36335182/93104683-16751f80-f6a6-11ea-8370-a27018ad1514.JPG)

**Comment by AnthonyWharton** [comment]
Ahh yes, you're just seeing the docs platform version number.  This is generated by the Read the Docs engine that we use and matches our backend API version number. We'll remove this from the page title now.

All of our applications (client/server) have their own version numbers which are managed on their respective platforms. We'll look into making a page that documents the latest versions of each on the docs in future, however it's fairly safe to trust updates from our official channels (linked from the website/docs) as all of these require code signing in order to be published.
